.github/workflows/ci.yml

```
@@ -22,10 +22,21 @@ jobs:
       uses: actions/checkout@v2
       with:
         submodules: true
-    - name: Build ${{ matrix.distro }} Docker image with ${{ matrix.compiler }}
+    - name: Prepare base image in Docker
+      run: docker build . -f utils/docker/Dockerfile.${{ matrix.distro }}
+                          --build-arg DAOS_DEPS_BUILD=no
+    - name: Build dependencies in Docker
+      run: docker build . -f utils/docker/Dockerfile.${{ matrix.distro }}
+                          --build-arg DEPS_JOBS=10 --build-arg DAOS_BUILD=no
+    - name: Build Docker image with ${{ matrix.compiler }}
       run: docker build . -f utils/docker/Dockerfile.${{ matrix.distro }}
                           --build-arg COMPILER=${{ matrix.compiler }}
-                          -t ${{ matrix.distro }}/${{ matrix.compiler }}/daos:${{ github.sha }}
+                          --build-arg DEPS_JOBS=10 --build-arg DAOS_JAVA_BUILD=no
```

Use the more natural boolean logic available in the Bourne shell:
```suggestion
                          --build-arg DEPS_JOBS=10 --build-arg DAOS_JAVA_BUILD=false
```